packages/test/test-end-to-end-tests/src/test/gc/gcSweepAttachmentBlobs.spec.ts

```diff
@@ -1341,7 +1349,16 @@ describeCompat("GC attachment blob sweep tests", "NoCompat", (getTestObjectProvi
 
 					// Load a container from the above summary, process all ops (including any GC ops) and validate that
 					// the deleted blob cannot be retrieved.
-					const container2 = await loadContainer(summary.summaryVersion);
+					// We load with GC Disabled to confirm that the GC Op is processed regardless of such settings
+					const config_gcSweepDisabled = JSON.parse(
+						JSON.stringify(testContainerConfig),
+					) as ITestContainerConfig;
+					// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+					config_gcSweepDisabled.runtimeOptions!.gcOptions!.enableGCSweep = undefined;
+					const container2 = await loadContainer(
+						summary.summaryVersion,
```

Comment:
This loads from the summary after or before the sweep op? If I understand things correctly, the summary is generated before the sweep op

Reply:
Yeah my point was to have the GC Op be processed by a client with Sweep disabled.  Your question about loading a new client from the summary that omits the deleted stuff is covered elsewhere I think, I'm going to try to find it.

Reply:
Hmm you may be right, I can't find that case.  Opened AB#7158 for it.  Resolving these comments since it's generic feedback on Sweep, not specific to the logic changes here.